# Array of object detections
std_msgs/Header header
perception/ObjectDetection[] detections